provision/setup-automatic-updates

```diff
@@ -36,9 +38,9 @@ if [ "$VAULT_AUTH_GITHUB_TOKEN" = "" ]; then
     export VAULT_AUTH_GITHUB_TOKEN=${token}
 fi
 vault login -method=github
-SLACK_WEBHOOK=$(vault read -field=password secret/slack/autoupdate-webhook)
+SLACK_WEBHOOK=$(vault read -field=value secret/slack/$CHANNEL)
```

Comment:
You can use a single webhook for multiple channels. So rather than thinking of a hook as a per-channel thing, I think it's best to think of it as a per-task thing.  Here this should always be the `autoupdate-hook` key from vault.  But _always_ get the `monitor-hook` element from vault and then use the payload to switch channel by adding `\"channel\": \"#montagu-test\"` (note the `#`) - use `$CHANNEL` here to change between `montagu-test` and `montagu-monitor`

Reply:
Updated - so I now always read secret/slack/monitor-webhook, and in the payload, I add \"channel\": \"#$CHANNEL\" into the payload - where $CHANNEL is an environment variable (or default montagu-monitor), supplied at the time of *installation*.